src/app/cart/cart.component.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../styles.scss';
+
+.modal{
```

Comment:
Can you please use http://getbem.com/introduction/ for the CSS?

Reply:
Please call this `cart-modal` if it's very cart specific...

Reply:
this is bootstrap card with modal class and here I am overriding its style.